README.md

```diff
@@ -180,6 +181,12 @@ If you lose your AVB or OTA signing key, you will no longer be able to sign new
 
 7. Reboot back into fastboot and lock the bootloader. This will trigger a data wipe again.
 
+    ```bash
+   fastboot flashing lock
```

Comment:
There's 1 missing space here for the indentation, but otherwise, this looks good to me. Thanks!

Reply:
Good catch :monocle_face:  :+1: